src/test/java/stepDefinitions/LoginStepDefs.java

```diff
@@ -0,0 +1,62 @@
+package stepDefinitions;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import pages.LoginPage;
+import pages.MainPage;
+
+import java.time.Duration;
+
+public class LoginStepDefs {
+    WebDriver driver;
+    WebDriverWait wait;
+
+    @Before //("I open browser")
+    public void iOpenBrowser(){
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
```

Comment:
After the Chrome 111 update, you can no longer kick off a chromedriver instance unless you add an additional chrome option:
"--remote-allow-origins=*"

 ChromeOptions options = new ChromeOptions();
        options.addArguments("--disable-notifications","--remote-allow-origins=*", "--incognito","--start-maximized");
        driver = new ChromeDriver(options);


Below are the list of available and most commonly used arguments for ChromeOptions class

**start-maximized:** Opens Chrome in maximize mode
**incognito:** Opens Chrome in incognito mode
**headless:** Opens Chrome in headless mode
**disable-extensions:** Disables existing extensions on Chrome browser
**disable-popup-blocking:** Disables pop-ups displayed on Chrome browser
**make-default-browser:** Makes Chrome default browser
**version:** Prints chrome browser version
**disable-infobars:** Prevents Chrome from displaying the notification ‘Chrome is being controlled by automated software

For example

![image](https://user-images.githubusercontent.com/104761482/225715669-8df3a92c-0c87-49f8-a865-31c6bc830996.png)

Reply:
I added the Chrome driver options as you mentioned. Please review. Thanks for your help!!!

Reply:
Hello

Please, re-submit homework 27 for review

Reply:
I pushed and commit my last updates. GitHub rejected it for some reason. Hope it works this time.